<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>patientInfo</class>
 <widget class="QWidget" name="patientInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>568</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>879</width>
    <height>568</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>879</width>
    <height>568</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>个人信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* 主窗口样式 */
QWidget {
    font-family: "Segoe UI", "Microsoft YaHei", sans-serif;
    background-color: #f8fafc;
}

/* 标签样式 */
QLabel {
    background-color: transparent;
    color: #374151;
    font-size: 14px;
    font-weight: 500;
}

/* 标题标签样式 */
#label_7 {
    font-size: 24px;
    font-weight: 600;
    color: #1f2937;
    padding: 8px 0px;
}

/* 输入框样式 */
QLineEdit {
    background-color: white;
    border: 1px solid #e2e8f0;
    border-radius: 8px;
    padding: 10px 12px;
    font-size: 14px;
    color: #1f2937;
    selection-background-color: #4a90e2;
    selection-color: white;
    min-height: 30px;
    max-width: 200px;
}

QLineEdit:focus {
    border: 1px solid #4a90e2;
    background-color: #f0f7ff;
    outline: none;
}

QLineEdit:hover {
    border: 1px solid #cbd5e1;
}

QLineEdit:read-only {
    background-color: #f1f5f9;
    color: #64748b;
    border: 1px solid #e2e8f0;
}

/* 按钮样式 */
QPushButton {
    background-color: #4a90e2;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 12px 20px;
    font-size: 14px;
    font-weight: 500;
    min-width: 80px;
    min-height: 40px;
    transition: all 0.2s ease;
}

QPushButton:hover {
    background-color: #3a7bc8;
    transform: translateY(-1px);
}

QPushButton:pressed {
    background-color: #2c6bb6;
    transform: translateY(1px);
}

QPushButton:disabled {
    background-color: #cbd5e1;
    color: #94a3b8;
}

/* 表单容器样式 */
#gridLayoutWidget {
    background-color: white;
    border-radius: 12px;
    border: 1px solid #e2e8f0;
    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.05);
}

/* 按钮容器样式 */
#horizontalLayoutWidget {
    background-color: transparent;
}

/* 布局样式 */
QGridLayout {
    margin: 20px;
    spacing: 15px;
}

QVBoxLayout, QHBoxLayout {
    spacing: 10px;
}

/* 图片样式 */
QLabel[scaledContents="true"] {
    border-radius: 8px;
    opacity: 0.1;
}
   </string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>691</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>出生日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>手机号</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>身份证号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>邮箱</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="birthLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phoneLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="genderLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="editPushButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="savePushButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>个人信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-20</y>
     <width>61</width>
     <height>611</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources.qrc">:/resources/icons/1注册登录d.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
